From 10c7e67e11a56e2fe1acf9b085772bc995d35bc0 Mon Sep 17 00:00:00 2001
From: Alexandre Rostovtsev <tetromino@gentoo.org>
Date: Sat, 7 Apr 2012 17:57:36 -0400
Subject: [PATCH] Fix glib includes for building with >=glib-2.31

glib-2.31 and newer no longer allow most glib subheaders to be included
directly.

https://savannah.nongnu.org/bugs/index.php?36127
---
 nautilus-gksu/libnautilus-gksu.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/nautilus-gksu/libnautilus-gksu.c b/nautilus-gksu/libnautilus-gksu.c
index 8e44d29..4acf3f8 100644
--- a/nautilus-gksu/libnautilus-gksu.c
+++ b/nautilus-gksu/libnautilus-gksu.c
@@ -5,7 +5,7 @@
 #include <string.h>
 #include <pthread.h>
 
-#include <glib/gkeyfile.h>
+#include <glib.h>
 #include <gtk/gtk.h>
 #include <gio/gio.h>
 #include <libnautilus-extension/nautilus-extension-types.h>
-- 
1.7.8.5

